src/features/areaAssignments/l10n/messageIds.ts

```diff
@@ -78,42 +75,44 @@ export default makeMessages('feat.areaAssignments', {
         label: m('Assignees'),
         unassigned: m('Only unassigned areas'),
       },
+      header: m('Filters'),
       title: m('Add filters to decide what areas you see on the map'),
       ungroupedTags: m('Ungrouped tags'),
     },
     findArea: {
       filterPlaceHolder: m('Filter'),
-      title: m('Find area'),
+      title: m('Areas'),
     },
     mapStyle: {
       area: {
-        label: m('What the area color represents.'),
+        label: m('Area color'),
         options: {
-          assignees: m('Number of assignees'),
+          assignees: m('Assignees'),
           hidden: m('Hidden'),
-          households: m('Number of households'),
+          households: m('Households'),
           outlined: m('Outlined'),
-          progress: m('Progress in this assignment'),
+          progress: m('Progress'),
         },
       },
       center: {
-        label: m('What to show in the center of the area'),
+        label: m('Area markers'),
         options: {
           assignees: m('Assignees'),
           hidden: m('Hidden'),
-          households: m('Number of locations and households in the area'),
-          progress: m('Progress in the area in this assignment'),
+          households: m('Households & locations'),
+          progress: m('Progress'),
         },
       },
       markers: {
-        label: m('What the markers represent'),
+        label: m('Locations markers'),
         options: {
-          dot: m('Dot'),
+          dot: m('Point'),
           hidden: m('Hidden'),
-          households: m('Number of households at the location'),
-          progress: m('Progress in this assignment'),
+          households: m('Households'),
+          progress: m('Progress'),
         },
       },
+      title: m('Layers'),
```

Comment:
"Layers" was decided to be a not-great idea, then changed to "map styles" which was also decided to be a not great idea. Either new evidence has surfaced that makes "layers" a great idea again, or we need a better name title for this part :)

Reply:
Asking to the team to see if we can find a better fit! :)

Reply:
Going with `Data visualization` after team feedback